ui/app/adapters/sync/association.js

```diff
@@ -24,6 +22,19 @@ export default class SyncAssociationAdapter extends ApplicationAdapter {
     return this.ajax(url, 'GET');
   }
 
+  // fetch sync status by destination for kv v2 secret details view
+  fetchDestinations({ mount, secretName }) {
```

Comment:
I wonder if we're mixing up concerns here since this is returning destinations? If so, this might be better in the destinations adapter. I realize that the url contains associations and that's confusing but if we are loading destination models from the response we should move this. Right now the `namespace` in the destination adapter is `v1/sys`. I think we could match it to this adapter and use `v1/sys/sync` and then there is the flexibility to use this different endpoints in either adapter depending on what data is returned.

Reply:
That sounds good to me - I feel like when I started working here adapters were more about similar request paths but they're really more about managing ember data models...if we move the polling to the component then I wonder if this even needs to be defined in the adapter. We aren't using a model here so would it be clearer if we just did `adapter.ajax('/v1/sys/sync/associations/:mount/:path, 'GET')` in the component